docs/use-cases-actors.md

```diff
@@ -0,0 +1,23 @@
+---
+id: use-cases-actors
+title: Scalable Actors
+sidebar_label: Scalable Actors
+---
+
+## Motivation
+
+A typical pattern is to have a workflow instance per business entity:
+
+- A workflow that tracks the status of a single IoT device.
+- A loyalty program that accumulates reward points per customer.
+- A routine that manages a unique resource in a conflict-free manner.
+
+Each flow responds to asynchronous events from a target entity, persists some corresponding state, and takes actions according to the defined rules.
+
+This programming paradigm is commonly known as the Actor Model.
+
+## Actors in Temporal
+
+Temporal workflows are suitable to implement scalable actor systems. A workflow execution represents a single actor, uses signals for events, and automatically keeps track of the state using the backend service.
```

Comment:
May be something that emphasizes the value of Temporal comparing to other existing Actor solutions.

Reply:
Yes, that would be great. The actors use case is the least detailed because I haven't found any data in the customer case studies that I saw. If you have any "from the trenches" stories of how and why people choose Temporal for actors, please share offline. cc @rylandg